languageserver/integration/entrypoint.go

```diff
@@ -33,6 +34,8 @@ const (
 	entryPointKindUnknown entryPointKind = iota
 	entryPointKindScript
 	entryPointKindTransaction
+	entryPointKindContract
+	entryPointKindContractInterface
```

Comment:
Contracts and contract interfaces are not entry points in the same sense as transactions and scripts. 

Maybe parse the signers for the contract (interface) in a new function instead of adding that to the entry point parsing function

Reply:
Got it. Will refactor it.